independent-projects/tools/devtools-common/src/main/java/io/quarkus/platform/catalog/processor/CatalogProcessor.java

```diff
@@ -22,6 +26,10 @@ public static CatalogProcessor of(ExtensionCatalog catalog) {
         final Map<String, List<Extension>> extsByCategory = new HashMap<>(catalog.getCategories().size());
         for (Extension e : catalog.getExtensions()) {
             List<String> categories = ExtensionProcessor.of(e).getCategories();
+            if (categories.isEmpty()) {
+                extsByCategory.put(UNCATEGORIZED_ID, new ArrayList<>());
```

Comment:
This seems to discard existing uncategorized extensions, is that intended?

Reply:
@gastaldi why would it discard them?

Reply:
it's the opposite, it's making them visible

Reply:
Well, it's in a loop, It seems like the `extsByCategory` map would already have an entry in the second iteration, but I might have been missing something

Reply:
Ah snap, I missed that, I thought it was adding to the categories list 😬. I will push a patch now

Reply:
https://github.com/quarkusio/quarkus/pull/21522